nmostesting/suites/IS0703Test.py

```diff
@@ -0,0 +1,97 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..GenericTest import GenericTest
+from ..IS05Utils import IS05Utils
+from ..IS07Utils import IS07Utils
+
+EVENTS_API_KEY = "events"
+CONN_API_KEY = "connection"
+
+
+class IS0703Test(GenericTest):
+    """
+    Runs IS-07-03-Test
+    """
+    def __init__(self, apis):
+        GenericTest.__init__(self, apis)
+        self.events_url = self.apis[EVENTS_API_KEY]["url"]
+        self.connection_url = self.apis[CONN_API_KEY]["url"]
+        self.is05_utils = IS05Utils(self.connection_url)
+        self.is07_utils = IS07Utils(self.events_url)
+
+    def set_up_tests(self):
+        self.senders = self.is05_utils.get_senders()
+        self.sources = self.is07_utils.get_sources_states_and_types()
+        self.transport_types = {}
+        self.sender_active_params = {}
+
+        for sender in self.senders:
+            if self.is05_utils.compare_api_version(self.apis[CONN_API_KEY]["version"], "v1.1") >= 0:
+                self.transport_types[sender] = self.is05_utils.get_transporttype(sender, "sender")
+            else:
+                self.transport_types[sender] = "urn:x-nmos:transport:rtp"
+
+        if len(self.senders) > 0:
+            for sender in self.senders:
+                dest = "single/senders/" + sender + "/active"
+                valid, response = self.is05_utils.checkCleanRequestJSON("GET", dest)
+                if valid:
+                    if len(response) > 0 and isinstance(response["transport_params"][0], dict):
+                        self.sender_active_params[sender] = response["transport_params"][0]
+
+    def test_01(self, test):
+        """Each Sender has the required ext parameters"""
+
+        ext_params_websocket = ['ext_is_07_source_id', 'ext_is_07_rest_api_url']
+        ext_params_mqtt = ['ext_is_07_rest_api_url']
+        if len(self.senders) > 0:
+            for sender in self.senders:
+                if sender in self.sender_active_params:
+                    all_params = self.sender_active_params[sender].keys()
+                    params = [param for param in all_params if param.startswith("ext_")]
+                    if (self.transport_types[sender] == "urn:x-nmos:transport:websocket" or 
+                        self.transport_types[sender] == "urn:x-nmos:transport:mqtt"):
+                        valid_params = False
+                        if self.transport_types[sender] == "urn:x-nmos:transport:websocket":
+                            if sorted(params) == sorted(ext_params_websocket):
+                                valid_params = True
+                        elif self.transport_types[sender] == "urn:x-nmos:transport:mqtt":
+                            if sorted(params) == sorted(ext_params_mqtt):
+                                valid_params = True
+                        if not valid_params:
+                            return test.FAIL("Missing required ext parameters")
```

Comment:
Might be useful to indicate the affected Sender here to assist with debug:
```suggestion
                            return test.FAIL("Missing required ext parameters for Sender {}".format(sender))
```

Reply:
Yep will make the change.